spec/features/admin/documentation_spec.rb

```diff
@@ -7,7 +7,9 @@
   before do
     login_as user
     doc.summary_content = 'original summary content'
+    doc.page_title = 'original page title'
```

Comment:
could you refactor these to use a `let!` for creating the doc?

Reply:
👍